Bucketeer/Sources/Public/BKTConfig.swift

```diff
@@ -1,6 +1,6 @@
 import UIKit
 
-public struct BKTConfig {
+public class BKTConfig {
```

Comment:
`BKConfig` 
- It has a property named `logger` which could be an instance of a class.
So It should be a class. Because a Struct's property should be only basic data_type, not a class.

- Next, we put a `Builder` class as an inner class here and we want to use the `Swift convenience init` to reuse the base `init` method of `BKConfig`, and `Builder` will not know too much about how to init the outer class.

Reply:
fixed by remove the breaking changes